src/Serilog.Sinks.AzureDataExplorer.Tests/AzureDataExplorerDurableSinkE2ETests.cs

```diff
@@ -0,0 +1,265 @@
+using Kusto.Data;
+using Kusto.Data.Net.Client;
+using Kusto.Data.Common;
+using Serilog.Sinks.AzureDataExplorer.Extensions;
+
+namespace Serilog.Sinks.AzureDataExplorer.Tests;
+
+/*
+ * This is an End to End Testcase which requires the following input to be set as environment variables
+ * ingestionURI : ingestion URL of ADX
+ * databaseName : database name 
+ * bufferFileName : If durable mode is required, we need to mention the buffer file name
+ * appId : Application client Id
+ * appKey : Application client key
+ * tenant : Authority
+ * These E2E testcases are disabled by default,
+ * to enable it
+ * 1. please change the access specifier of this class (AzureDataExplorerDurableSinkE2ETests) to public
+ * 2. remove the System.Diagnostics.CodeAnalysis.SuppressMessage
+ */
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1000:Test classes must be public", Justification = "Disabled")]
```

Comment:
Would it be better that we see if these env vars are present and if so , we can run the test else log a warning that the environment variables need to be present for the tests to run ?
This is just a thought , as this will mean that users do not have to change these modifiers in run . Also , when we run E2E tests in the pipeline , we dont have to do this . Else for future PR's we will need to run these tests manually

log.Warning("E2E tests need the following variables to be set - Tests will be skipped")

Reply:
added skippable for this E2Etests